ios/browser/api/networking/common/BUILD.gn

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//build/config/ios/rules.gni")
+import("//ios/build/config.gni")
+
+source_set("common") {
```

Comment:
`common` doesn't make sense inside `ios/browser`. `common` is used to for things that are shared between `browser`, `renderer`, etc... so putting it inside `browser` confuses the convention I think